docs/UserGuide.md

```diff
@@ -325,18 +350,14 @@ Closes a claim from an insurance plan of a client.
 
 Format: `closeClaim INDEX iid/INSURANCE_ID cid/CLAIM_ID`
 
-IDs for insurance plans:
```

Comment:
Is it better to keep this ID reference for insurance plans in each command? Or will it be too repetitive

Reply:
i'll merge first, if it seems too repetitive, will remove in a future PR